lib/formats/base64.flow

```diff
@@ -9,9 +9,6 @@ export {
 
 	// Convert binary (using lower 8 bits of each character) to base-64 encoded string
```

Comment:
Do add a comment explaining that if you need this to work with Unicode, then use expandUtf8 from utf8.flow to help the next person